test/parallel/test-child-process-spawnsync-input.js

```diff
@@ -76,10 +76,9 @@ let options = {
   input: 1234
 };
 
-assert.throws(function() {
-  spawnSync('cat', [], options);
-}, /TypeError:.*should be Buffer, Uint8Array or string not number/);
-
+common.expectsError(
+  () => spawnSync('cat', [], options),
+  { code: 'ERR_INVALID_ARG_TYPE', type: TypeError }, 1);
```

Comment:
Just making an observation. This is the only place in this patch where `expectsError` is explicitly passed `1`.